.github/workflows/ci-Instrumentation.Owin.yml

```diff
@@ -0,0 +1,17 @@
+name: Build OpenTelemetry.Instrumentation.Owin
+
+on:
+  pull_request:
+    branches: [ 'main*', 'instrumentation*' ]
+    paths:
+    - '*/OpenTelemetry.Instrumentation.Owin*/**'
+    - 'build/**'
+    - '!**.md'
```

Comment:
I ahve missed it earlier, whats about path to Shared code?

Reply:
Good call! I added `src/Shared/**` to all the dedicated workflows so far.